css-view-transitions-1/Overview.bs

```diff
@@ -1189,6 +1221,27 @@ urlPrefix: https://wicg.github.io/navigation-api/; type: interface;
 			* If |callbackPromise| was rejected with reason |r|, then [=reject=] |transition|'s [=ViewTransition/DOM updated promise=] with |r|.
 	</div>
 
+## [=Clear view transition=] ## {#clear-view-transition-algorithm}
+
+	<div algorithm>
+		To <dfn>clear view transition</dfn> of a {{ViewTransition}} |transition|:
+
+		1. Let |document| be |transition|'s [=relevant global object's=] [=associated document=].
+
+		1. Remove all associated [=view-transition pseudo-elements=] from |document|.
+
+				Issue: There needs to be a definition/link for "remove".
+
+				Issue: There needs to be a definition/link for "associated".
+
+		1. [=list/For each=] |capturedElement| of |transition|'s [=ViewTransition/named elements=]' [=map/values=]:
+
+			1. [=list/For each=] |style| of |capturedElement|'s [=captured element/style definitions=]:
+
+				1. If |style| is not null,
+					then remove |style| from |document|'s [=document/view transition style sheet=].
```

Comment:
We were missing this bit before, so styles would just hang around forever.

Reply:
I like that idea, but wouldn't it mean a user style like:

```css
::view-transition {
  content: '';
}
```

…would cause the view transition pseudo to be 'always there'?

Reply:
I've filed https://github.com/w3c/csswg-drafts/issues/8113 to think more about this.